examples/England-MedicationRequest-BundleDMPrescriptionType-Example.xml

```diff
@@ -0,0 +1,388 @@
+<Bundle xmlns="http://hl7.org/fhir">
+    <id value="England-MedicationRequest-BundleDMPrescriptionType-Example" />
```

Comment:
ID: England-Bundle-MedicationPrescriptionType-Example
Example name should be prefixed after resource type, in this case, the resource type is Bundle